model-archiver/README.md

```diff
@@ -2,34 +2,33 @@
 
 ## Contents of this Document
 * [Overview](#overview)
-* [MMS model-archiver CLI](#model-archiver-command-line-interface)
+* [model-archiver CLI](#model-archiver-command-line-interface)
 * [Artifact Details](#artifact-details)
     * [MAR-INFO](#mar-inf)
     * [Model name](#model-name)
     * [Runtime](#runtime)
     * [Handler](#handler)
-* [Dependent and Nested models](#dependent-and-nested-models)
 
 ## Other Relevant Documents
-* [Model Archive Example](../examples/mxnet_vision.md)
-* [Packaging an ONNX Model](convert_from_onnx.md)
+* [Model Archive Example](../examples/mxnet_vision/README.md)
+* [Packaging an ONNX Model](docs/convert_from_onnx.md)
 
 ## Overview
 
 A key feature of MMS is the ability to package all model artifacts into a single model archive file. It is a separate command line interface (CLI), `model-archiver`, that can take model checkpoints and package them into a `.mar` file that can then be redistributed and served by anyone using MMS. It takes in the following model artifacts: a model composed of one or more files, the description of the model's inputs in the form of a signature file, a service file describing how to handle inputs and outputs, and other optional assets that may be required to serve the model. The CLI creates a `.mar` file that MMS's server CLI uses to serve the models.
 
-**Important**: Make sure you try the [Quick Start: Export a Model](../README.md#model-archive) tutorial for a short example of using `model-archiver`.
+**Important**: Make sure you try the [Quick Start: Creating a Model Archive](../README.md#create-a-model-archive) tutorial for a short example of using `model-archiver`.
 
 MMS support arbitrary models file. It's custom service code's responsibility to locate and load models files. Following information are required to create a standalone model archive:
 1. [Model name](#model-name)
 2. [Model path](#model-path)
 3. [Handler](#handler)
 
-## MMS Export Command Line Interface
+## model-archiver Command Line Interface
```

Comment:
Model Archiver (starting with capital might be good.

Reply:
done